Modules/BenchPress.Azure/Tests/Public/CustomAssertions/BeDeployed.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+using module ./../../../Classes/ConfirmResult.psm1
+
```

Comment:
Add

```
BeforeAll {
  . $PSScriptRoot/../../../Public/CustomAssertions/BeDeployed.ps1
}
```
to get the unit tests to pass.

(same to the other files with the respective path)

Reply:
Adding this fails it locally on my machine;

`Assertion operator name 'BeDeployed' has been added multiple times.`

I'll see what happens when I push it up to be run on the agent

Reply:
I have to forcibly remove the pester module and it worked locally